lms/static/sass/discussion/views/_thread.scss

```diff
@@ -106,24 +72,124 @@ body.discussion, .discussion-module {
     }
 
     .comment-actions-list {
+      @include float(right);
```

Comment:
Hmm, floating and inline-block+vertical align middle. Usually its one method or the other. Are both needed here?